lib/page/performMouseAction.js

```diff
@@ -30,6 +30,20 @@ const performMouseAction = async function performMouseAction(context, actionType
     return ({ status: STATUS.INVISIBLE });
   }
 
+  const overlappingNode = await context.page.evaluate(selectorNode => {
+    selectorNode.scrollIntoView({ block: 'center' });
```

Comment:
Not sure it's fine to always trigger scrolling even when we don't need it. Scrolling may cause side-effects we don't want to get 🤔

Reply:
Also we're scrolling vertically only, is it fine?

Reply:
I rechecked this code and found out that this scroll is redundant.